robottelo/cli/base.py

```diff
@@ -112,8 +138,8 @@ def create(cls, options=None):
             cls._construct_command(options), output_format='csv')
 
         # Extract new object ID if it was successfully created
-        if len(result.stdout) > 0 and 'id' in result.stdout[0]:
-            obj_id = result.stdout[0]['id']
+        if len(result) > 0 and 'id' in result[0]:
```

Comment:
It was originally `if len(something) > 0` so I left it with the same style.
Regarding your question - I guess it depends on type of "`something`". If "`something`" was, for example, string or dictionary - then i guess there's no point in verifying it's length. But `result` here is a list. And for `list` object, imo, it adds readability. For me `if len(list) > 0` == 'if list is not empty and has entries' and it's a little bit clearer than `if list`.
Anyway, it's definitely not an issue and i can update it if needed.